java-client/src/generated/java/org/opensearch/client/opensearch/OpenSearchAsyncClient.java

```diff
@@ -61,6 +65,12 @@ public OpenSearchAsyncClient withTransportOptions(@Nullable TransportOptions tra
         return new OpenSearchAsyncClient(this.transport, transportOptions);
     }
 
+    // ----- Child clients
+
+    public OpenSearchDanglingIndicesAsyncClient danglingIndices() {
```

Comment:
Why it was moved from `OpenSearchXxxClientBase` class?

Reply:
Because it's now generated it's moved to the generated version of the client class

Reply:
Hm ... the `OpenSearchXxxClientBase` classes are public so we cannot just remove methods from them I believe for 2.x at least. We could keep the  `OpenSearchXxxClientBase` unchanged, right? (since it will be just an override in this case).

Reply:
The `*Base` classes only exist in main, and haven't been backported to `2.x` so we haven't actually locked into its public contract yet

Reply:
Ah got it, thanks @Xtansia !